CustomerJourney/src/[sandbox]/sandbox.mock.ts

```diff
@@ -63,3 +63,120 @@ export const mockedTimelineItems: any = [
     timestamp: "2021-03-05T19:00:05.596Z"
   }
 ];
+
+export const sampleTemplate = {
+  Name: "Test Template 2",
+  DatapointCount: 1000,
+  Attributes: [
+    {
+      Version: "0.1",
+      Event: "Quote",
+      Metadata: "email",
+      DisplayName: "Email",
+      AggregationMode: "Value",
+      type: "tab",
+      tag: "email",
+      Limit: 1
+    },
+    {
+      Version: "0.1",
+      Event: "Quote",
+      Metadata: "Make",
+      DisplayName: "Make",
+      AggregationMode: "Value",
+      Limit: 1,
+      type: "table"
+    },
+    {
+      Version: "0.1",
+      Event: "Quote",
+      Metadata: "Model",
+      DisplayName: "Model",
+      Limit: 1,
+      AggregationMode: "Value",
+      type: "table"
+    },
+    {
+      Version: "0.1",
+      Event: "Quote",
+      Metadata: "License Plate",
+      DisplayName: "License Plate",
+      Limit: 1,
+      AggregationMode: "Value",
+      type: "table"
+    },
+    {
+      Version: "0.1",
+      Event: "Quote",
+      Metadata: "ltv",
+      DisplayName: "LTV",
+      AggregationMode: "Value",
+      Limit: 1,
+      type: "table",
+      // eslint-disable-next-line prettier/prettier
+      formatValue: (val: string) => {
+        if (val) {
+          return `$${val.match(/\d\d\d/g)?.join(",")}`;
+        }
+
+        return val;
+      }
+    },
+    {
+      Version: "0.1",
+      Event: "Quote",
+      Metadata: "firstName",
+      DisplayName: "First Name",
+      Limit: 1,
+      AggregationMode: "Value",
+      tag: "name",
+      type: "inline"
+    },
+    {
+      Version: "0.1",
+      Event: "Quote",
```

Comment:
Hm . . . In reality this sample template ais not actually used anywhere . ..  but we should have it be more clear what are the specific standars that we neeed to adhere by . . . because whatever shape we give here _needs_ to be understood by the component downstream. For example, I spent a lot of time debugging undefined values because the had different Capitalizatiion . . . . seems brittle in that sense